lib/blocks.js

```diff
@@ -5,11 +5,17 @@ var async = require('async');
 var bitcore = require('bitcore-lib');
 var pools = require('../pools.json');
 var BN = bitcore.crypto.BN;
+var LRU = require('lru-cache');
+
 
 function BlockController(node) {
   var self = this;
   this.node = node;
 
+  this.blockSummaryCache = LRU(1000000);
+  this.blockCacheConfirmations = 6;
+  this.blockCache = LRU(1000);
```

Comment:
Added config options in: https://github.com/bitpay/insight-api/pull/467/commits/4e54d195b947e1ac023d365e967b572b453c954b